### ISSUE TITLE
Can't get anything to download

### ISSUE THREAD

**Issue by thecozmicfool** [evilhero/mylar#2193]
Basically, my issue is that I can't get anything to download. It's obvious from the logs that it's searching for the files but it never can find anything, despite me being able to find the files when search for them myself on my indexers.  I'm a total noob with this stuff so I won't be surprised if it turns out to be some dumb mistake on my part. I just can't figure it our for myself. I'm sorry if this is something that's already been resolved. I tried searching through the current issues but couldn't find anything that fixed my issue.  Thanks for your time!

[mylar.log](https://github.com/evilhero/mylar/files/2849725/mylar.log)

**Comment by barbequesauce** [comment]
Looking at that log, it seems you’re doing alright - you’re fixing problems as they come up, so good for you!

That said, traceback are not good (TM). html5lib seems to be missing. Try running

`pip install html5lib —upgrade
`

And then restart Mylar...

**Comment by thecozmicfool** [comment]
Haha I did as much as I could using google and various forums but I got totally stuck at this point. Thanks so much for the help! That seems to have fixed my issues. Sabnzbd and qbittorrent are giving me some issues now but at least Mylar is finding the files and sending them to the proper programs. I’m sure there are just some tweaks I need to make in the settings to get everything running smoothly. Thanks again for your help!
